HelpSource/Reference/Assignment.schelp

```diff
@@ -14,6 +14,50 @@ x = [1, 2, 3, 4].rotate(1);
 c = a + b;
 ::
 
+section:: Assignment in Declaration
+
+When declaring a variable, a value can be assigned to it immediately:
+
+code::
+(
+var x = 1;
+var y = -1;
+x + y
+)
+::
+
+Such assignments can be chained up:
+code::
+(
+var x = 1;
+var y = x + 9;
+x + y
+)
+::
+
+section:: Reassignment
+
+A variable can be given a new value of any kind at any time.
```

Comment:
I would say this whole section can probably be reduced to a single sentence + examples: "A variable can be assigned a value at any time, including during declaration"

```supercollider
var foo = 3;
foo = nil;
```

Also might be worth saying something like "A variable can be reassigned a value of any class, even if the previous value was of a different class."